client/plots/singleCellPlot.js

```diff
@@ -120,13 +120,13 @@ class singleCellPlot {
 	async main() {
 		this.config = JSON.parse(JSON.stringify(this.state.config))
 
-		this.table.selectAll('*').remove()
+		this.dom.tableDiv.selectAll('*').remove()
 		this.plots = []
 		copyMerge(this.settings, this.config.settings.singleCellPlot)
 
-		this.headerTr = this.table.append('tr')
-		this.tr = this.table.append('tr')
 		let sample = this.state.config.sample
+		this.legendRendered = false
+
 		if (this.state.dslabel != 'GDC') {
```

Comment:
can these be replaced by this?
body.sample= this.state.config.file || this.state.config.sample

Reply:
No, this is a boolean to render the legend only for the first plot

Reply:
Is updated now to use the variable this.tableOnPlot that it is initialized as 
`		this.tableOnPlot = appState.nav?.header_mode == 'hidden'
`